builder/azure/arm/config_test.go

```diff
@@ -15,16 +15,13 @@ import (
 var requiredConfigValues = []string{
 	"capture_name_prefix",
 	"capture_container_name",
-	"client_id",
```

Comment:
I'm not sure why these were required before, especially given the use of a managed identity.